frontend/src/component/common/Card/Card.tsx

```diff
@@ -58,30 +126,28 @@ const StyledCardActions = styled(Box)(({ theme }) => ({
 }));
 
 const StyledCardBodyContent = styled(Box)(({ theme }) => ({
-    color: theme.palette.text.secondary,
-    fontSize: theme.fontSizes.smallBody,
     marginTop: theme.spacing(2),
 }));
 
-const StyledCardFooter = styled(Box)(({ theme }) => ({
+const StyledCardFooter = styled(Box, {
+    shouldForwardProp: (prop) => prop !== 'cardVariant',
+})<{ cardVariant: CardVariant }>(({ theme, cardVariant }) => ({
     padding: theme.spacing(0, 2),
     display: 'flex',
-    background: theme.palette.background.elevation1,
-    boxShadow: theme.boxShadows.accordionFooter,
     alignItems: 'center',
     justifyContent: 'space-between',
-    borderTop: `1px solid ${theme.palette.divider}`,
     minHeight: theme.spacing(6.25),
     fontSize: theme.fontSizes.smallerBody,
-    color: theme.palette.text.secondary,
     textWrap: 'nowrap',
+    ...variants[cardVariant].footer(theme),
 }));
 
-interface ICardProps {
+interface ICardProps extends Omit<CardProps, 'title'> {
```

Comment:
Since we're extending from CardProps anyways, want to go the extra step and make this component [stylable](https://github.com/Unleash/unleash/pull/9604) in this PR? That way you can close mine as this one would be superseding.

Reply:
Incorporated here [43c37eb](https://github.com/Unleash/unleash/pull/9614/commits/43c37eb207f141bc84b2eb44a1d73e1e85df20ce)